_utilities/layout-grid.md

```diff
@@ -30,6 +28,63 @@ subnav:
   href: '#sass-mixins'
 - text: Latest updates
   href: '#changelog'
+spacing_settings:
+  - name: Small grid gap
+    subsection: grid-gap token
+    section: true
+    description: Sets `"grid-gap-sm"` token.
```

Comment:
I believe the token is just called "sm"

```suggestion
    description: Sets the `.grid-gap-sm` utility class and the `"sm"` grid gap token.
```

Reply:
Removed in [5b6e4da](https://github.com/uswds/uswds-site/pull/2276/commits/5b6e4da238ec15b95c66f14f23af8b3dab049315)